### ISSUE TITLE
Mastodon: Almost empty body in received comments

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#6829]
### Example 1

Mastodon post: https://framapiaf.org/@DelphineM/101715514662784848
Friendica post: https://friendica.mrpetovan.com/display/22b2c6df6cbecd898580abc306f0913939d61619

Source object:
```json
{
	"@context": ["https://www.w3.org/ns/activitystreams", "https://w3id.org/security/v1", {
		"manuallyApprovesFollowers": "as:manuallyApprovesFollowers",
		"sensitive": "as:sensitive",
		"movedTo": {
			"@id": "as:movedTo",
			"@type": "@id"
		},
		"alsoKnownAs": {
			"@id": "as:alsoKnownAs",
			"@type": "@id"
		},
		"Hashtag": "as:Hashtag",
		"ostatus": "http://ostatus.org#",
		"atomUri": "ostatus:atomUri",
		"inReplyToAtomUri": "ostatus:inReplyToAtomUri",
		"conversation": "ostatus:conversation",
		"toot": "http://joinmastodon.org/ns#",
		"Emoji": "toot:Emoji",
		"focalPoint": {
			"@container": "@list",
			"@id": "toot:focalPoint"
		},
		"featured": {
			"@id": "toot:featured",
			"@type": "@id"
		},
		"schema": "http://schema.org#",
		"PropertyValue": "schema:PropertyValue",
		"value": "schema:value"
	}],
	"id": "https://framapiaf.org/users/DelphineM/statuses/101715514662784848/activity",
	"type": "Create",
	"actor": "https://framapiaf.org/users/DelphineM",
	"published": "2019-03-08T14:33:02Z",
	"to": ["https://www.w3.org/ns/activitystreams#Public"],
	"cc": ["https://framapiaf.org/users/DelphineM/followers", "https://mastodon.xyz/users/nextcloud", "https://framapiaf.org/users/Carnet", "https://framapiaf.org/users/shonpolo"],
	"object": {
		"id": "https://framapiaf.org/users/DelphineM/statuses/101715514662784848",
		"type": "Note",
		"summary": null,
		"inReplyTo": "https://framapiaf.org/users/shonpolo/statuses/101715155697928700",
		"published": "2019-03-08T14:33:02Z",
		"url": "https://framapiaf.org/@DelphineM/101715514662784848",
		"attributedTo": "https://framapiaf.org/users/DelphineM",
		"to": ["https://www.w3.org/ns/activitystreams#Public"],
		"cc": ["https://framapiaf.org/users/DelphineM/followers", "https://mastodon.xyz/users/nextcloud", "https://framapiaf.org/users/Carnet", "https://framapiaf.org/users/shonpolo"],
		"sensitive": false,
		"atomUri": "https://framapiaf.org/users/DelphineM/statuses/101715514662784848",
		"inReplyToAtomUri": "https://framapiaf.org/users/shonpolo/statuses/101715155697928700",
		"conversation": "tag:framapiaf.org,2019-02-19:objectId=12019911:objectType=Conversation",
		"content": "\u003cp\u003e\u003cspan class=\"h-card\"\u003e\u003ca href=\"https://framapiaf.org/@shonpolo\" class=\"u-url mention\"\u003e@\u003cspan\u003eshonpolo\u003c/span\u003e\u003c/a\u003e\u003c/span\u003e Mieux, je ne sais pas mais différent. Je vois \u0026quot;Carnet\u0026quot; comme des post-it alors que \u0026quot;Notes\u0026quot; serait plus pour des \u0026quot;longues\u0026quot; notes. Ex : je me promène dans la rue et je vois une affiche que je veux approfondir -\u0026gt; \u0026quot;Carnet\u0026quot;. Je suis en conférences et veux prendre des notes tout du long -\u0026gt; \u0026quot;Notes\u0026quot;. \u003cbr /\u003eNB : Carnet n\u0026apos;a d\u0026apos;intérêt, pour mon usage, qu\u0026apos;avec l\u0026apos;appli mobile. Pas d\u0026apos;édition possible des \u0026quot;Notes\u0026quot; depuis le mobile, si ? \u003cbr /\u003e\u003cspan class=\"h-card\"\u003e\u003ca href=\"https://framapiaf.org/@Carnet\" class=\"u-url mention\"\u003e@\u003cspan\u003eCarnet\u003c/span\u003e\u003c/a\u003e\u003c/span\u003e \u003cspan class=\"h-card\"\u003e\u003ca href=\"https://mastodon.xyz/@nextcloud\" class=\"u-url mention\"\u003e@\u003cspan\u003enextcloud\u003c/span\u003e\u003c/a\u003e\u003c/span\u003e\u003c/p\u003e",
		"contentMap": {
			"fr": "\u003cp\u003e\u003cspan class=\"h-card\"\u003e\u003ca href=\"https://framapiaf.org/@shonpolo\" class=\"u-url mention\"\u003e@\u003cspan\u003eshonpolo\u003c/span\u003e\u003c/a\u003e\u003c/span\u003e Mieux, je ne sais pas mais différent. Je vois \u0026quot;Carnet\u0026quot; comme des post-it alors que \u0026quot;Notes\u0026quot; serait plus pour des \u0026quot;longues\u0026quot; notes. Ex : je me promène dans la rue et je vois une affiche que je veux approfondir -\u0026gt; \u0026quot;Carnet\u0026quot;. Je suis en conférences et veux prendre des notes tout du long -\u0026gt; \u0026quot;Notes\u0026quot;. \u003cbr /\u003eNB : Carnet n\u0026apos;a d\u0026apos;intérêt, pour mon usage, qu\u0026apos;avec l\u0026apos;appli mobile. Pas d\u0026apos;édition possible des \u0026quot;Notes\u0026quot; depuis le mobile, si ? \u003cbr /\u003e\u003cspan class=\"h-card\"\u003e\u003ca href=\"https://framapiaf.org/@Carnet\" class=\"u-url mention\"\u003e@\u003cspan\u003eCarnet\u003c/span\u003e\u003c/a\u003e\u003c/span\u003e \u003cspan class=\"h-card\"\u003e\u003ca href=\"https://mastodon.xyz/@nextcloud\" class=\"u-url mention\"\u003e@\u003cspan\u003enextcloud\u003c/span\u003e\u003c/a\u003e\u003c/span\u003e\u003c/p\u003e"
		},
		"attachment": [],
		"tag": [{
			"type": "Mention",
			"href": "https://framapiaf.org/users/shonpolo",
			"name": "@shonpolo"
		}, {
			"type": "Mention",
			"href": "https://framapiaf.org/users/Carnet",
			"name": "@Carnet"
		}, {
			"type": "Mention",
			"href": "https://mastodon.xyz/users/nextcloud",
			"name": "@nextcloud@mastodon.xyz"
		}]
	},
	"signature": {
		"type": "RsaSignature2017",
		"creator": "https://framapiaf.org/users/DelphineM#main-key",
		"created": "2019-03-08T14:33:02Z",
		"signatureValue": "rA71YsPl10wea+yqUpr5U/eCMkafnLrsDHfH2016fx0IiyS0RsjLid1k4ityofbjWU+xSbGYwLmDJaO+USeB8xCkdt3ZR1toQyP4UjJFTRkUszCE7EzWuEM2WXBwxJ/dgPJoXv8Uc3aIr+mvct3EgumxTMcr6HhVFgogolsbKlx1jjJnPvFy17+rHpz/U4E627e0UdWvNtrhZa94QxX2a51iiHn5dsNqTSMk+gEkOGB7xq2b/8w8YRUhArcZlNyABJ7MJX3ONiRkN2snxpXzn+iBz4ttmSEg+5G7NKfj4gmDzLrOwkuXDLIwxK6rMTAMBnlfyPBps3i18QuMnM5FOA=="
	}
}
```

### Example 2

Mastodon post: https://framapiaf.org/@DelphineM/101716124114402763
Friendica post: https://friendica.mrpetovan.com/display/22b2c6df2cd5af760e11617a701d961a33d610cc

Source object:
```json
{
	"@context": ["https://www.w3.org/ns/activitystreams", "https://w3id.org/security/v1", {
		"manuallyApprovesFollowers": "as:manuallyApprovesFollowers",
		"sensitive": "as:sensitive",
		"movedTo": {
			"@id": "as:movedTo",
			"@type": "@id"
		},
		"alsoKnownAs": {
			"@id": "as:alsoKnownAs",
			"@type": "@id"
		},
		"Hashtag": "as:Hashtag",
		"ostatus": "http://ostatus.org#",
		"atomUri": "ostatus:atomUri",
		"inReplyToAtomUri": "ostatus:inReplyToAtomUri",
		"conversation": "ostatus:conversation",
		"toot": "http://joinmastodon.org/ns#",
		"Emoji": "toot:Emoji",
		"focalPoint": {
			"@container": "@list",
			"@id": "toot:focalPoint"
		},
		"featured": {
			"@id": "toot:featured",
			"@type": "@id"
		},
		"schema": "http://schema.org#",
		"PropertyValue": "schema:PropertyValue",
		"value": "schema:value"
	}],
	"id": "https://framapiaf.org/users/DelphineM/statuses/101716124114402763/activity",
	"type": "Create",
	"actor": "https://framapiaf.org/users/DelphineM",
	"published": "2019-03-08T17:08:02Z",
	"to": ["https://www.w3.org/ns/activitystreams#Public"],
	"cc": ["https://framapiaf.org/users/DelphineM/followers", "https://mastodon.xyz/users/nextcloud", "https://framapiaf.org/users/Carnet", "https://framapiaf.org/users/shonpolo"],
	"object": {
		"id": "https://framapiaf.org/users/DelphineM/statuses/101716124114402763",
		"type": "Note",
		"summary": null,
		"inReplyTo": "https://framapiaf.org/users/shonpolo/statuses/101715718850862333",
		"published": "2019-03-08T17:08:02Z",
		"url": "https://framapiaf.org/@DelphineM/101716124114402763",
		"attributedTo": "https://framapiaf.org/users/DelphineM",
		"to": ["https://www.w3.org/ns/activitystreams#Public"],
		"cc": ["https://framapiaf.org/users/DelphineM/followers", "https://mastodon.xyz/users/nextcloud", "https://framapiaf.org/users/Carnet", "https://framapiaf.org/users/shonpolo"],
		"sensitive": false,
		"atomUri": "https://framapiaf.org/users/DelphineM/statuses/101716124114402763",
		"inReplyToAtomUri": "https://framapiaf.org/users/shonpolo/statuses/101715718850862333",
		"conversation": "tag:framapiaf.org,2019-02-19:objectId=12019911:objectType=Conversation",
		"content": "\u003cp\u003e\u003cspan class=\"h-card\"\u003e\u003ca href=\"https://framapiaf.org/@shonpolo\" class=\"u-url mention\"\u003e@\u003cspan\u003eshonpolo\u003c/span\u003e\u003c/a\u003e\u003c/span\u003e Ah, je n\u0026apos;avais pas repéré l\u0026apos;appli Notes :D \u003cbr /\u003eÇa peut servir ;) \u003cbr /\u003e\u003cspan class=\"h-card\"\u003e\u003ca href=\"https://framapiaf.org/@Carnet\" class=\"u-url mention\"\u003e@\u003cspan\u003eCarnet\u003c/span\u003e\u003c/a\u003e\u003c/span\u003e \u003cspan class=\"h-card\"\u003e\u003ca href=\"https://mastodon.xyz/@nextcloud\" class=\"u-url mention\"\u003e@\u003cspan\u003enextcloud\u003c/span\u003e\u003c/a\u003e\u003c/span\u003e\u003c/p\u003e",
		"contentMap": {
			"en": "\u003cp\u003e\u003cspan class=\"h-card\"\u003e\u003ca href=\"https://framapiaf.org/@shonpolo\" class=\"u-url mention\"\u003e@\u003cspan\u003eshonpolo\u003c/span\u003e\u003c/a\u003e\u003c/span\u003e Ah, je n\u0026apos;avais pas repéré l\u0026apos;appli Notes :D \u003cbr /\u003eÇa peut servir ;) \u003cbr /\u003e\u003cspan class=\"h-card\"\u003e\u003ca href=\"https://framapiaf.org/@Carnet\" class=\"u-url mention\"\u003e@\u003cspan\u003eCarnet\u003c/span\u003e\u003c/a\u003e\u003c/span\u003e \u003cspan class=\"h-card\"\u003e\u003ca href=\"https://mastodon.xyz/@nextcloud\" class=\"u-url mention\"\u003e@\u003cspan\u003enextcloud\u003c/span\u003e\u003c/a\u003e\u003c/span\u003e\u003c/p\u003e"
		},
		"attachment": [],
		"tag": [{
			"type": "Mention",
			"href": "https://framapiaf.org/users/shonpolo",
			"name": "@shonpolo"
		}, {
			"type": "Mention",
			"href": "https://framapiaf.org/users/Carnet",
			"name": "@Carnet"
		}, {
			"type": "Mention",
			"href": "https://mastodon.xyz/users/nextcloud",
			"name": "@nextcloud@mastodon.xyz"
		}]
	},
	"signature": {
		"type": "RsaSignature2017",
		"creator": "https://framapiaf.org/users/DelphineM#main-key",
		"created": "2019-03-08T17:08:02Z",
		"signatureValue": "zhJJsNZ/dqmUIfKN2uObI8I9tECoO0CNbLbLp2dYdA4OaKRnNp8N8qglSSilCRDJ3Ka3lJR8LKnBLLOh18iqJrU7ui4dkVMhd8z2GnVcKB6RQosqmAJKlRYd8XhwBh62/FdU33F4evkEL+0i10Q1pfB4RZhvyjWAS6CrEVdGBhyY8jZswPilnXB5QiqaTye+jr/4CJ22tlotJZ/r/KuejnxQFz/qpOen5K3a0QS2YWwI55K6vMWLJwEVELhzwiurvoapfCS2DwfkyWuGBShpURIQhNY54qB/x3GYGDL7DBDHBAE/4kG8CYjvbOsxejJ34EkxN8hGqSmHtPticQxNzw=="
	}
}
```

**Comment by annando** [comment]
Could it be some regexp going wild? Remember that we had some similar issue before with the removing of mentions.

**Comment by MrPetovan** [comment]
Yes, this is a reminder to look into it, I posted it from work.

**Comment by MrPetovan** [comment]
Caused by #6649

**Comment by MrPetovan** [comment]
Closed by #6839
